src/templates/layouts/healthCareLocalFacility/index.tsx

```diff
@@ -143,7 +144,7 @@ export function HealthCareLocalFacility({
                       vaHealthConnectPhoneNumber={vaHealthConnectPhoneNumber}
                       fieldTelephone={fieldTelephone}
                     />
-                    <div>TODO: Office hours</div>
+                    <Hours allHours={officeHours} headerType="clinical" />
```

Comment:
[This is what `content-build` does.](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/layouts/health_care_local_facility.drupal.liquid#L132-L132)